.g8/yesNoPage/default.properties

```diff
@@ -1,2 +1,3 @@
 description = Generates a controller and view for a page with a single yes / no question
+package = returns
```

Comment:
this is the default, feels ok to use returns? if not feel free to overrule :)

Reply:
Still have to type `returns` in response to the question, i think this just puts suggestion text next to the prompt